boot/libs.ml

```diff
@@ -1,4 +1,4 @@
-let executables = [ "main" ]
+let executables = [ "main"; "man_page_gen" ]
```

Comment:
why do we need this change?

Reply:
I think bootstrapping did this automatically.